src/main/java/dev/openfga/sdk/api/OpenFgaApi.java

```diff
@@ -832,12 +969,18 @@ private CompletableFuture<ApiResponse<Void>> writeAssertions(
                 .replace("{store_id}", ApiClient.urlEncode(storeId.toString()))
                 .replace("{authorization_model_id}", ApiClient.urlEncode(authorizationModelId.toString()));
 
+        Map<Attribute, String> telemetryAttributes = new HashMap<Attribute, String>() {
+            {
+                put(Attributes.FGA_CLIENT_REQUEST_METHOD, "WriteAssertions");
+                put(Attributes.FGA_CLIENT_REQUEST_STORE_ID, storeId);
+                put(Attributes.FGA_CLIENT_REQUEST_MODEL_ID, authorizationModelId);
```

Comment:
I would prefer if you call a function which is not autogenerated to deduce these rather than do git patching on the generator. Git patching in the generator has caused a lot of maintenance work in the past and we are trying to get away from it